{"channeldata":{"activate.d":false,"binary_prefix":false,"deactivate.d":false,"description":null,"dev_url":null,"doc_source_url":null,"doc_url":null,"home":"https://github.com/pytest-dev/pytest-faulthandler","icon_hash":null,"icon_url":null,"identifiers":null,"keywords":null,"license":"MIT","post_link":false,"pre_link":false,"pre_unlink":false,"recipe_origin":null,"run_exports":{},"source_git_url":null,"source_url":"https://pypi.io/packages/source/p/pytest-faulthandler/pytest-faulthandler-2.0.1.tar.gz","subdirs":["win-64"],"summary":"py.test plugin that activates the fault handler module for tests (dummy package now)","tags":null,"text_prefix":false,"timestamp":1562279197,"version":"1.5.0"},"channeldata_version":1,"feedstock":null,"labels":["main"],"package":"pytest-faulthandler-1.5.0-py36_0.tar.bz2","repodata":{"arch":"x86_64","build":"py36_0","build_number":0,"constrains":["python_abi * *_cp36m"],"depends":["pytest","python 3.6*"],"license":"MIT","license_family":"MIT","md5":"72e5f779fa3ff878fb17aa5fa774e953","name":"pytest-faulthandler","platform":"win","sha256":"d9b63d00f6bb935a00af705a185855d7789ce34f61088552b0437e3229d22d01","size":9026,"subdir":"win-64","version":"1.5.0"},"repodata_version":1,"subdir":"win-64","url":"https://conda.anaconda.org/conda-forge/win-64/pytest-faulthandler-1.5.0-py36_0.tar.bz2"}